contracts/modules/capital/Ramm.sol

```diff
@@ -524,20 +541,12 @@ contract Ramm is IRamm, MasterAwareV2 {
 
     require(slot1.updatedAt == 0, "ALREADY_INITIALIZED");
```

Comment:
I think this needs to be an if condition, not a require statement, as I think we'll brick future updates otherwise